### PR TITLE
DIA-2525 fix no action wrong numbers in our reports

### PR DESCRIPTION
**Pull request by andresilveirah**

### CODE REVIEW

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt [hunk 1]
```diff
@@ -160,15 +154,6 @@ private class DataStorageCcpaImpl(context: Context) : DataStorageCcpa {
             .apply()
     }
 
-    override var ccpaPostChoiceResp: String?
```
Comment (bohdan-go-wombat) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt:163]: did you confirm that we are not using this value at all and we do not need to store it?

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageGdpr.kt [hunk 2]
```diff
@@ -208,15 +202,6 @@ private class DataStorageGdprImpl(context: Context) : DataStorageGdpr {
             .apply()
     }
 
-    override var gdprPostChoiceResp: String?
```
Comment (bohdan-go-wombat) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageGdpr.kt:211]: same here

**Review by bohdan-go-wombat — APPROVED** [review]

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt [hunk 3]
```diff
@@ -418,23 +418,9 @@ private class CampaignManagerImpl(
...
 
-    override var gdprMessageMetaData: MessageMetaData?
-        get() {
-            return dataStorage.gdprMessageMetaData?.let { JsonConverter.converter.decodeFromString<MessageMetaData>(it) }
-        }
-        set(value) {
-            val serialised = value?.let { JsonConverter.converter.encodeToString(value) }
-            dataStorage.gdprMessageMetaData = serialised
-        }
+    override var gdprMessageMetaData: MessageMetaData? = null
```
Comment (Nevazhnovu) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt:421]: Is this value set somewhere or it'll be forever `null` ?

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt [hunk 4]
```diff
@@ -418,23 +418,9 @@ private class CampaignManagerImpl(
...
 
-    override var ccpaMessageMetaData: MessageMetaData?
-        get() {
-            return dataStorage.ccpaMessageMetaData?.let { JsonConverter.converter.decodeFromString<MessageMetaData>(it) }
-        }
-        set(value) {
-            val serialised = value?.let { JsonConverter.converter.encodeToString(value) }
-            dataStorage.ccpaMessageMetaData = serialised
-        }
+    override var ccpaMessageMetaData: MessageMetaData? = null
```
Comment (Nevazhnovu) [inline cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt:423]: Is this value set somewhere or it'll be forever `null` ?

**Review by Nevazhnovu — APPROVED** [review]

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt (hunk 3 above)
Comment (andresilveirah) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt:421]: this value is set when the SDK receives the response of `/messages`

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt (hunk 4 above)
Comment (andresilveirah) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/campaign/CampaignManager.kt:423]: Same here.

File: cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt (hunk 1 above)
Comment (andresilveirah) [reply cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/local/DataStorageCcpa.kt:163]: Yep.

**Pull request merged**